### ISSUE TITLE
Support async/await syntax

### ISSUE THREAD

**Issue by wwilsman** [thefrontside/bigtest#62]
Based on the popularity of async/await usage in tests, supporting this without `.run()` would reduce the boilerplate required to use this pattern with `@bigtest/interaction` page objects.

**With `.run()`:**
``` javascript
it('does things', async () => {
  await page.fillName('name namerson').run();

  expect(page.name).to.equal('name namerson');

  await page.clickSubmit().run();

  expect(page.hasSuccessMessage).to.be.true;
});
```

**Proposed:**
``` javascript
it('does things', async () => {
  await page.fillName('name namerson');

  expect(page.name).to.equal('name namerson');

  await page.clickSubmit();

  expect(page.hasSuccessMessage).to.be.true;
});
```

**Comment by wwilsman** [comment]
We could also maybe support auto-wrapping getters in a convergence in this case. That would enable us to use convergences with the same async/await syntax.

``` javascript
it('would require chai-as-promised', async () => {
  await page.fillName('name namerson');
  await expect(page.name).to.eventually.equal('name namerson');
  await page.clickSubmit();
  await expect(page.hasSuccessMessage).to.eventually.be.true;
});
```

**Comment by Robdel12** [comment]
Closed with #77
